### PR TITLE
new cheat sheet (terminal commands) for IPFS

### PR DESCRIPTION
**Pull request by fzzr-**
IA: https://duck.co/ia/view/ipfs_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@fzzr- Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

**Comment by fzzr-** [comment]
I think I should remove "Installing" section because there terrible long path depending version.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=IPFS Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/ipfs_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/ipfs.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
+{
+  "id": "ipfs_cheat_sheet",
+  "name": "IPFS",
+  "description": "A quick reference to IPFS, a peer-to-peer hypermedia protocol for The Permanent Web.\nThe InterPlanetary File System (IPFS) is a new hypermedia distribution protocol, addressed by content and identities. IPFS enables the creation of completely distributed applications. It aims to make the web faster, safer, and more open.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/ipfs.json]: Let's shorten this. It should be a really succint description of the cheat sheet content. Also we don't support `\n` in this property. Only in the `key` and `val`

**Comment by moollaza** [comment]
@fzzr- Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@fzzr- this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

File: share/goodie/cheat_sheets/json/ipfs.json (hunk 1 above)
Comment (fzzr-) [inline share/goodie/cheat_sheets/json/ipfs.json]: I'll do it. This should be a separate pull-request (because this already merged)?

**Comment by fzzr-** [comment]
Thanks!

File: share/goodie/cheat_sheets/json/ipfs.json (hunk 1 above)
Comment (fzzr-) [inline share/goodie/cheat_sheets/json/ipfs.json]: fixed in https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2895

**Pull request merged**